cfgov/jinja2/v1/_includes/molecules/image-text-50-50.html

```diff
@@ -70,7 +69,7 @@
             {% endif %}
 
             <img src="{{ photo.url }}"
-                 alt="{{ photo.alt }}"
+                 alt="{{ photo.title }}"
```

Comment:
this is true, i'll actually remove it because users dont want to add an alt on the cms end.